server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
some more digging found that the Sensu service Handler() method is being called however the Handle() method is not.  It is called if the .sensu() is in the tick script. How does the topic handler know to use the sensu service?